src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -67,6 +70,11 @@
     public static final String MONIKER_ONE = "moniker one";
     public static final String MONIKER_TWO = "moniker two";
     public static final String MONIKER_THREE = "moniker three";
+    public static final String OBJECT_URN_QUERY_PARAMS = "objectUrnQueryParams";
```

Comment:
a couple more constants.  I tried to think of better names that would represent what these are used for but I could not come up with anything better